deliciousmods/1956_beta
# Fixed Ulan Bataar Conquest and Expanded Tuva. I can't boot the game so check thoroughly

To test:
Conquer Ulan Bataar with another country and/or trigger the event cityfall.103.t

I expanded the Tuvan focus tree. I used the visualization tool and nothing else. There may be overlapping icons because I can't see on the same screen the loaded branches. There may also be potentially balance problems, although it's Tuva and while they are certainly a bit stronger, I don't think that it changes that much. 


Suggestion:
They could potentially have from some kind of army debuff  concluded with the creation of the ministry of war though. It could also be tied with military production which was... superficial IRL

To fix independently, as I said to Marijn Tuva has the unique privilege to be able to form two unions: Siberia and Turkestan. Logically they should only be able to form only one of them. It also mean that they should have access to the two shared post unification trees but can only do one of them. (some dlc condition shenanigans may also apply).
